docs/backup-restore.md

```diff
@@ -90,7 +90,8 @@ Specifically, please follow the steps below.
    * Remove table A
    * Create a table named A with backup B
 
-Note that tables can only be restored one by one so you need to do the above steps for each table.
+Note:
 
-Note also that some of the configurations such as PITR and auto scaling policies are reset to the default values for restored tables, so please manually configure required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
-If you used the schema loader and didn't change any other configurations, executing the schema loader with the same schema file is one of the ways to restore the original configuration. It only sets missing configurations if a specified table exists.
+* You can create a backup for the `scalardb.metadata` table without restoring the PITR because it contains metadata of the schemas (It will not change after the schema creation).
+* Tables can only be restored one by one so you need to do the above steps for each table.
+* Configurations such as PITR and auto scaling policies are reset to the default values for restored tables, so please manually configure the required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
```

Comment:
```suggestion
* Configurations such as PITR and auto-scaling policies are reset to the default values for restored tables, so please manually configure required settings. See [the doc](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CreateBackup.html#CreateBackup_HowItWorks-restore) for more detail.
```

Reply:
I will integrate.

Reply:
Above suggestion integrated [6604cc9](https://github.com/scalar-labs/scalardb/pull/571/commits/6604cc90402e1e6505d99b4fb1f5f2289e71442f)

Thank you for your suggestion.